lib/floe/workflow/states/task.rb

```diff
@@ -79,12 +82,43 @@ def end?
 
         def validate_state!(workflow)
           validate_state_next!(workflow)
+          validate_state_timeout_seconds!(workflow)
+          validate_state_timeout_seconds_path!(workflow)
+        end
+
+        def validate_state_timeout_seconds!(workflow)
+          return if @timeout_seconds.nil?
+          return if @timeout_seconds.kind_of?(Integer) && @timeout_seconds > 0
+
+          invalid_field_error!("TimeoutSeconds", @timeout_seconds, "must be positive, non-zero integer")
```

Comment:
NOTE positive, non-zero integer is redundant but this is the exact wording that is in the states-language spec